src/entities/activity/ui/items/TextItem.tsx

```diff
@@ -11,10 +17,15 @@ type TextItemProps = {
 
 export const TextItem = ({ item, value, onValueChange, isDisabled }: TextItemProps) => {
   const { maxResponseLength } = item.config;
+  const [hasError, setHasError] = useState(false);
```

Comment:
Should be initialized with the proper value
![CleanShot 2024-09-03 at 10 57 52@2x](https://github.com/user-attachments/assets/73a516a4-1ae3-4ef7-ab6d-7600bebbe69f)

Reply:
It's showing 304/300 (error) in a default color variant (meaning no error). This was after I moved to the next question and then back